### ISSUE TITLE
Updating to Python 3.9 results in 'HTMLParser' object has no attribute 'unescape' error

### ISSUE THREAD

**Issue by aggi3000** [novoid/Memacs#105]
After updating to Python 3.9 on Arch I get the attribute error `'HTMLParser' object has no attribute 'unescape'` when evoking a memacs module:

> ERROR:root:Traceback (most recent call last):
     File "/home/user/.local/lib/python3.9/site-packages/memacs/lib/memacs.py", line 188, in handle_main
       self._main()
     File "/home/user/.local/lib/python3.9/site-packages/memacs/sms.py", line 270, in _main
       xml.sax.parseString(data.encode('utf-8'),
     File "/usr/lib/python3.9/xml/sax/__init__.py", line 48, in parseString
       parser.parse(inpsrc)
     File "/usr/lib/python3.9/xml/sax/expatreader.py", line 111, in parse
       xmlreader.IncrementalParser.parse(self, source)
     File "/usr/lib/python3.9/xml/sax/xmlreader.py", line 125, in parse
       self.feed(buffer)
     File "/usr/lib/python3.9/xml/sax/expatreader.py", line 217, in feed
       self._parser.Parse(data, isFinal)
     File "/build/python/src/Python-3.9.1/Modules/pyexpat.c", line 407, in StartElement
     File "/usr/lib/python3.9/xml/sax/expatreader.py", line 333, in start_element
       self._cont_handler.startElement(name, AttributesImpl(attrs))
     File "/home/user/.local/lib/python3.9/site-packages/memacs/sms.py", line 158, in startElement
       sms_body = htmlparser.unescape(sms_body.replace('EnCoDiNgHaCk42', '&#'))
   AttributeError: 'HTMLParser' object has no attribute 'unescape'
Traceback (most recent call last):
  File "/home/user/.local/bin/memacs_sms", line 8, in <module>
    sys.exit(main())
  File "/home/user/.local/lib/python3.9/site-packages/bin/memacs_sms.py", line 34, in main
    memacs.handle_main()
  File "/home/user/.local/lib/python3.9/site-packages/memacs/lib/memacs.py", line 188, in handle_main
    self._main()
  File "/home/user/.local/lib/python3.9/site-packages/memacs/sms.py", line 270, in _main
    xml.sax.parseString(data.encode('utf-8'),
  File "/usr/lib/python3.9/xml/sax/__init__.py", line 48, in parseString
    parser.parse(inpsrc)
  File "/usr/lib/python3.9/xml/sax/expatreader.py", line 111, in parse
    xmlreader.IncrementalParser.parse(self, source)
  File "/usr/lib/python3.9/xml/sax/xmlreader.py", line 125, in parse
    self.feed(buffer)
  File "/usr/lib/python3.9/xml/sax/expatreader.py", line 217, in feed
    self._parser.Parse(data, isFinal)
  File "/build/python/src/Python-3.9.1/Modules/pyexpat.c", line 407, in StartElement
  File "/usr/lib/python3.9/xml/sax/expatreader.py", line 333, in start_element
    self._cont_handler.startElement(name, AttributesImpl(attrs))
  File "/home/user/.local/lib/python3.9/site-packages/memacs/sms.py", line 158, in startElement
    sms_body = htmlparser.unescape(sms_body.replace('EnCoDiNgHaCk42', '&#'))
AttributeError: 'HTMLParser' object has no attribute 'unescape'

As far as I can tell the python 3.9 version of HTMLParser dropped unescape. But there should be a workaround by running memacs in a 3.8-env for now.

**Comment by novoid** [comment]
Yes, it got deprecated: https://docs.python.org/3/whatsnew/3.9.html#porting-to-python-3-9 -> scrolling up a bit shows:

> The unescape() method in the html.parser.HTMLParser class has been removed (it was deprecated since Python 3.4). html.unescape() should be used for converting character references to the corresponding unicode characters.

@aggi3000: since I'm not running Python 3.9 on my hosts to confirm it in your environment: are you feeling confident on trying to move to `html.unescape()` and sending a patch?

**Comment by aggi3000** [comment]
I'm confident enough to try editing the source on my machine but not confident enough to send a git patch... :(

What I did: In sms.py I replaced

>  `##` reverse encoding hack from just before:
>                sms_body = htmlparser.unescape(sms_body.replace('EnCoDiNgHaCk42', '&#'))

with 

> `##` reverse encoding hack from just before:
>               sms_body = html.unescape(sms_body.replace('EnCoDiNgHaCk42', '&#'))

Everything seems to be working fine with Python 3.9 but I didn't manage to get a 3.8 environment running simultaneously so I don't know if that change would break anything. Shame on me that I don't know git enough to make a patch out of this. I'm sorry... But maybe that confirmation at least helps a bit?
